std/algorithm/iteration.d

```diff
@@ -2228,7 +2286,8 @@ if (isForwardRange!Range)
  * all elements in a given subrange are equivalent under the given predicate.
  * With a unary predicate, a range of tuples is returned, with the tuple
  * consisting of the result of the unary predicate for each subrange, and the
- * subrange itself.
+ * subrange itself. The range is currently a reference range, but this may
+ * change in the future.
```

Comment:
"Reference range" could be read as meaning either that the range's elements are references, or that the range itself is a reference. Probably better to be more specific; something like, "copies of the range currently have reference semantics".